packages/plugin-commands-init/src/init.ts

```diff
@@ -56,6 +56,9 @@ export async function handler (
   }
   const config = await parseRawConfig(opts.rawConfig)
   const packageJson = { ...manifest, ...config }
+  if (opts.initPackageManager) {
```

Comment:
There is a problem with the logic here. Currently, `opt.initPackageManager` is always `undefined`, and the value of the `init-package-manager` configuration cannot be obtained. The value of `config.packageManager` is `true/false`, which causes the value of the `packageManager` field in package.json to be true or false. @zkochan

Reply:
how is it always undefined if I set the default value to `true` in the config package?

Reply:
This is the fix: https://github.com/pnpm/pnpm/pull/9104